mslib/mscolab/file_manager.py

```diff
@@ -130,6 +154,19 @@ def is_collaborator(self, u_id, op_id):
             return False
         return True
 
+    def is_viewer(self, u_id, op_id):
+        """
+        op_id: operation id
+        u_id: user-id
+        """
+        # return true only if the user is viewer
+        permi = Permission.query.filter_by(u_id=u_id, op_id=op_id).first()
```

Comment:
please use  perm = 

so that someone reading this on some place will know Permission is meant